common/upload.js

```diff
@@ -106,22 +106,25 @@
             vm.erred = false;
 
             vm.totalSize = 0;
-            vm.humanTotalSize = 0
-            vm.noOfFiles = 0;
+            vm.humanTotalSize = 0;
+            vm.filesCt = 0;
+            vm.filesToUploadCt = 0;
             vm.sizeTransferred = 0;
             vm.humanSizeTransferred = 0;
 
             vm.checksumProgress = 0;
             vm.checksumCompleted = 0;
 
+            vm.preFileExistsCount = 0;
+
             vm.createUploadJobProgress = 0;
             vm.createUploadJobCompleted = 0;
 
             vm.fileExistsProgress = 0;
             vm.fileExistsCompleted = 0;
 
             vm.uploadProgress = 0;
-            vm.uploadCompleted = 0;
+            vm.numUploadsCompleted = 0;
```

Comment:
renamed this because there was a boolean with the same name attached to the `uploadFile` object.